CHANGES.rst

```diff
@@ -1,6 +1,11 @@
 Changes
 =======
 
+Version 0.3.1 (2018-09-07)
+--------------------------
+- Renames parameter to turn caching on or off to CACHE.
+- Adds to the documentation the user guide and getting started sections.
```

Comment:
Shall we also add "support workflow parameter interpolation"? from fc8205c.

Reply:
Good point, I wrote it as 'adds parameter passing to workflow steps'. It could be described better but it's not coming to me